packages/components/src/components/helper-text/helper-text.css

```diff
@@ -0,0 +1,36 @@
+.helper-text {
+  display: flex;
+  align-items: center;
+  flex-direction: row-reverse;
+  justify-content: flex-end;
+  font: var(--telekom-text-style-small-bold);
```

Comment:
I would add a scoped var (e.g. `--font` in the :host) for this one.

Reply:
also added color variables